swatch-producer-azure/src/main/resources/application.properties

```
@@ -43,8 +43,9 @@ AZURE_MARKETPLACE_BASE_URL=https://marketplaceapi.microsoft.com
 AZURE_MARKETPLACE_API_VERSION=2018-08-31
 AZURE_OIDC_SAAS_MARKETPLACE_RESOURCE=20e940b3-4c77-4b0b-9a53-9e16a1b010a7
 
+# wiremock defaults
 %wiremock.SWATCH_CONTRACTS_ENDPOINT=http://${WIREMOCK_HOSTNAME}:8101/mock/internalSub
-%ephemeral.SWATCH_CONTRACTS_ENDPOINT=${WIREMOCK_ENDPOINT}/mock/contractsApi
+# %ephemeral.SWATCH_CONTRACTS_ENDPOINT=${WIREMOCK_ENDPOINT}/mock/contractsApi
```

We depend on this for some component testing so ideally will keep this in.

There is a mismatch between here and wiremock proxy which would prevent it from getting proxied: https://github.com/RedHatInsights/wiremock-consoledot/blob/5f3b7f481f4fa5e6c8f6e2fa4fda437a6a175674/deploy/clowdapp.yaml#L129

```suggestion
%ephemeral.SWATCH_CONTRACTS_ENDPOINT=${WIREMOCK_ENDPOINT}/mock/contractApi
```

---

I've re-added and would like @liwalker-rh double check that the SSL issue is gone. If not, this is definetively caused by wiremock being in the middle and not accepting the SSL certificate.